scripts/mkdist.js

```diff
@@ -14,43 +14,40 @@ function mkdist (opts, cb) {
 
   var dist = opts.dist
   var ver = dist.version
-  if (ver.match(/^v/)) ver = ver.substring(1)
 
-  dist.releaseLink = 'releases/' + dist.id + '/v' + ver
-  var releasePath = opts.path + '/' + dist.releaseLink
-  dist.platforms = []
+  dist.releaseLink = dist.id + '/' + ver
+  var releasePath = opts.path + '/releases/' + dist.releaseLink
+  dist.platforms = {}
```

Comment:
Because Orbit is the only app using mkdist.js, and it is stale in comparison to the JSON object built by `build-go.sh`. I had to change a few things to make it automatically add the archs we had; this was one of them.